docs/reference/query-dsl/script-score-query.asciidoc

```diff
@@ -35,27 +36,58 @@ GET /_search
 --------------------------------------------------
 // CONSOLE
 
-NOTE: The values returned from `script_score` cannot be negative. In general,
-Lucene requires the scores produced by queries to be non-negative in order to
-support certain search optimizations.
 
-==== Accessing the score of a document within a script
+[[script-score-top-level-params]]
+==== Top-level parameters for `script_score`
+`query`::
+(Required, query object) Query used to return documents.
+
+`script`::
++
+--
+(Required, <<modules-scripting-using,script object>>) Script used to modify
+the <<relevance-scores,relevance score>> of documents returned by the `query`.
```

Comment:
Script used to modify the <<relevance-scores,relevance score>> -> Script used to compute the score of documents

Reply:
Fixed with 9028d83857f27699e62f7b7fa011b3f403059a43.